pkg/events/events.go

```diff
@@ -0,0 +1,34 @@
+package events
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+// Event message format "medik8s <operator shortname> <message>"
+const format = "medik8s %3s %s"
```

Comment:
I think I would skip "medik8s", it's not that helpful, especially downsytream. And I would somehow separate operator and message. Maybe `"%3s: %s"`? Or `"[%3s] %s"`?

Reply:
The idea of "medik8s" string was to help filtering our events

> Maybe "%3s: %s"? Or "[%3s] %s"?

agreed on this